src/Propel/Generator/Behavior/I18n/I18nBehavior.php

```diff
@@ -232,8 +279,10 @@ protected function addLocaleColumnToI18n()
 
     /**
      * Moves i18n columns from the main table to the i18n table
+     *
+     * @return void
      */
-    protected function moveI18nColumns()
+    protected function moveI18nColumns(): void
```

Comment:
Revert?

Reply:
Are we planning to not use void but $this? Otherwise we can keep it

Reply:
I will remove it for now, and we can add it in next release.